### ISSUE TITLE
hide_edge_borders smart option

### ISSUE THREAD

**Issue by bssb** [mboughaba/i3config.vim#18]
The following config line is highlighted in red:

```hide_edge_borders smart```

because the smart keyword is not recognized. This looks like it would be a quick fix. I will be happy to submit a PR for it.

**Comment by mboughaba** [comment]
Thanks for opening the ticket. Indeed smart keyword is a support keyword.

https://i3wm.org/docs/userguide.html#configuring

```
4.12. Hiding borders adjacent to the screen edges
hide_edge_borders none|vertical|horizontal|both|smart
```

**Comment by mboughaba** [comment]
fixed by #19
